src/screens/Account/EmptyStateAccount.js

```diff
@@ -26,6 +26,8 @@ class EmptyStateAccount extends PureComponent<{
   render() {
     const { account, parentAccount } = this.props;
     const mainAccount = getMainAccount(account, parentAccount);
+    const isEthereum = mainAccount.currency.ticker === "ETH";
```

Comment:
I think, same as with the previous comment, this needs to come from whether the currency has tokens or not, which is available from `account.tokenAccounts` being undefined or not, it should be empty/filled for a libcore eth and undefined for example for btc

Reply:
Thank you! I was looking for the right way to do this, didn't feel right at all